webpack.config.js

```diff
@@ -1,343 +1,382 @@
-const { join, relative, resolve, sep, dirname } = require("path");
+const { join, relative, resolve, sep, dirname } = require('path');
```

Comment:
Is there any changes buried in here or is it just autoformatting?

Reply:
Yes, I added the includePaths to scss loader